src/OpenApi/Generate.elm

```diff
@@ -1612,7 +1743,7 @@ operationToContentSchema qualify operation =
             case OpenApi.Reference.toConcrete requestOrRef of
                 Just request ->
                     OpenApi.RequestBody.content request
-                        |> contentToContentSchema qualify
+                        |> contentToContentSchema True
```

Comment:
Can't wait for the new elm-pages version to drop so we can completely remove `qualify`